### PR TITLE
Fds 1796 schema convert export pickle

### PR DESCRIPTION
**Pull request by afwillia**
The intent of this PR is to allow the `convert` CLI command to export the graph. It creates a new option, `output_type` with the values `jsonld`, `graph`, or `all`. `graph` exports the graph as a pickle, `jsonld` exports schema as a jsonld, and `all` does both.

Updated the CLI tests to ensure the file written by `convert` is read correctly.

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -46,6 +48,8 @@ def assert_expected_file(self, result, output_path):
 
     def test_schema_convert_cli(self, runner, helpers):
         data_model_csv_path = helpers.get_data_path("example.model.csv")
+        data_model_pickle_path = helpers.get_data_path("example.model.pickle")
```
Comment (mialy-defelice) [inline tests/test_cli.py:51]: For these I would try to find a way to supply these parameters via parmeterize, you can send the models via one parameter and the output type parameter via another. 
Then you could call the appropriate runner, based on your current parameters, do some checks to see where you care and call the appropriate asserts. Does that make sense?

**Review by mialy-defelice — CHANGES_REQUESTED** [review]
Looks good so far, I requested some changes in testing and  I will test for function tomorrow :)

**Comment by mialy-defelice** [comment]
@afwillia Also note your tests are failing only bc of black needing to be run on a couple files, if you run black and re-commit, it should get further :)

File: tests/test_cli.py (hunk 1 above)
Comment (afwillia) [reply tests/test_cli.py:51]: yeah, this makes sense. I'll play around with it!

**Pull request closed without merging**